### PR TITLE
custom error pages

### PR DESCRIPTION
**Pull request by anweshknayak**
@deshraj review please. all the changes you requested ( https://github.com/Cloud-CV/EvalAI/pull/283)

### CODE REVIEW

File: settings/common.py [hunk 1]
```diff
@@ -21,6 +21,12 @@
 
 sys.path.append(APPS_DIR)
 
+TEMPLATE_PATH = os.path.join(BASE_DIR, 'frontend')
+TEMPLATE_DIRS = (
+
```
Comment (deshraj) [inline settings/common.py]: Can you remove this line?

File: evalai/urls.py [hunk 2]
```diff
@@ -24,6 +24,9 @@
 
 from accounts.views import ConfirmEmailView
 
+handler404 = 'evalai.views.page_not_found'
+handler500 = 'evalai.views.server_error'
```
Comment (deshraj) [inline evalai/urls.py]: Can you please change the name of handler500 to `evalai.views.internal_server_error`

File: evalai/urls.py [hunk 3]
```diff
@@ -66,9 +69,9 @@
 # DJANGO-SPAGHETTI-AND-MEATBALLS URLs available during development only.
 if settings.DEBUG:
     urlpatterns += [url(r'^dbschema/',
-                    include('django_spaghetti.urls')),
+                        include('django_spaghetti.urls')),
                     url(r'^docs/',
-                    include('rest_framework_docs.urls')),
+                        include('rest_framework_docs.urls')),
```
Comment (deshraj) [inline evalai/urls.py]: Irrelevent changes for this PR. Please undo this. same for line 72.

Comment (anweshknayak) [inline evalai/urls.py]: sorry it is due to the autopep8 fix for flake error. Will fix

**Comment by anweshknayak** [comment]
@deshraj done review please

**Comment by deshraj** [comment]
Everything looks good except for the tests. It would be good if you can add the tests in the same PR. We can keep this PR open till then.

**Comment by anweshknayak** [comment]
@RishabhJain2018 is working on this. as discussed in #280

**Comment by deshraj** [comment]
Ok cool. I am keeping this PR open until the tests are done.

**Comment by RishabhJain2018** [comment]
@deshraj As the PR #314 is merged, I think we should remove the `error` folder from this PR ?

**Comment by deshraj** [comment]
@RishabhJain2018 syncing this PR with the latest code will automatically fix this issue.

**Comment by RishabhJain2018** [comment]
@deshraj I think it will create a problem because I have created a folder with the name `error-pages` and here it is `error`. so , it will create two folders for error.

**Comment by deshraj** [comment]
I see. Then go ahead and remove this folder.

**Comment by RishabhJain2018** [comment]
@anweshknayak Can you please update the PR with the `error` folder and the files `404.html` and `500.html` removed ?

**Comment by anweshknayak** [comment]
sure.

**Comment by anweshknayak** [comment]
custom page for error 500 is not there. shall I create one ?

**Comment by RishabhJain2018** [comment]
@anweshknayak Don't worry, I will create that.

**Comment by deshraj** [comment]
Closing it since already resolved in #329

**Pull request closed without merging**